__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -2039,14 +2079,14 @@ describe('meltTokens', () => {
 
   it('should melt tokens', async () => {
     const hWallet = await generateWalletHelper();
-    await GenesisWalletHelper.injectFunds(hWallet, await hWallet.getAddressAtIndex(0), 10);
+    await GenesisWalletHelper.injectFunds(hWallet, await hWallet.getAddressAtIndex(0), 15);
 
     // Creating the token
     const { hash: tokenUid } = await createTokenHelper(
       hWallet,
       'Token to Melt',
       'TMELT',
-      300,
+      500,
```

Comment:
We have 2 new requests that melt 100 each, so we need to mint more tokens before melting them